android/api.md

```diff
@@ -594,6 +609,252 @@ private static String MyOIDCAuthProvider(){
 }
 ```
 
+#### Multi-Auth
+
+This section talks about the capability of AWS AppSync to configure multiple authorization modes for a single AWS AppSync endpoint and region. Follow the [AWS AppSync Multi-Auth](https://docs.aws.amazon.com/appsync/latest/devguide/security.html#using-additional-authorization-modes) to configure multiple authorization modes for your AWS AppSync endpoint.
+
+As discussed in the above linked documentation, certain fields may be protected by different authorization types. This can lead the same query, mutation, or subscription to have different responses based on the authorization sent with the request; therefore, it is recommended to use different `AWSAppSyncClient` objects for each authorization type. Instantiation of multiple `AWSAppSyncClient` objects is enabled by passing `true` to the `useClientDatabasePrefix` flag. The `awsconfiguration.json` generated by the AWS AppSync console and Amplify CLI will add an entry into the `AppSync` section called `ClientDatabasePrefix`. This will be used to separate the caches used for operations such as queries, mutations, and subscriptions.
+
+**Important Note:** If you are an existing customer of AWS AppSync SDK for Android, the `useClientDatabasePrefix` has a default value of `false`.  When or if you choose to use multiple `AWSAppSyncClient` objects, turning on `useClientDatabasePrefix` will change the location of the caches used by the client. The caches will not be automatically moved; you are responsible for the migration of any data within the caches that you wish to keep and deleting the old cache on the device.
```

Comment:
"When or if you choose" --> "If you choose"

Reply:
Changed